README.md

```diff
@@ -197,15 +205,23 @@ class ExampleHook(BaseHook):
         }
  ```
 
```

Comment:
Overall, I think this section needs more information about what's going on in the code sample. The points below feel a bit random.